files/checks/check_systemd_service.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+
+[ -n "$2" ] || exit 1;
```

Comment:
According to the [docs](https://nagios-plugins.org/doc/guidelines.html#AEN78) the correct return code here would be 3.